socorro/siglists/README.rst

```diff
@@ -0,0 +1,195 @@
+Signatures Utilities Lists
```

Comment:
I ran this through pandoc to do the first pass of reformatting. Then I went through it and updated some stuff.

Making sure it's correct is good, so if you see anything that's not right, we should fix it in this PR.